IbisLib/filereader.cpp

```diff
@@ -85,6 +86,36 @@ QString FindExecutable( QStringList & candidatePaths )
 
 bool FileReader::FindMincConverter()
 {
+    if( !m_mincconvert.isEmpty() && !m_minccalc.isEmpty() )
+        return true;
+    // First check if there is a configuration file
+    QString mincconfig = QDir::homePath() + "/" + IBIS_CONFIGURATION_SUBDIRECTORY + "/minctoolspath.txt";
```

Comment:
Call Application::GetConfigDirectory() instead

Reply:
I think that if the file is present, it should overwrite the list of search paths and not be appended.

Reply:
fileReader does not know application